package/rm2fb/rm2fb.service

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+[Unit]
+Description=Remarkable2 Framebuffer Server
+Before=xochitl.service
```

Comment:
i don't think `Before=` actually causes xochitl to start, it is just a hint how to enable both services if they are both started at same time. (i think)

`Requires=` will cause xochitl (or the listed service) to start (if i understand systemd properly), but it will start them both at same time (hence the Before/After keywords)

Reply:
If it's a hint, likely it should be marked as before the launchers as well?